shared/attributes/failure_status_code.yml

```diff
@@ -0,0 +1,20 @@
+type: integer
+
+enum:
+  - 401
+  - 403
+  - 404
+  - 422
+  - 429
+  - 500
+
+description: >
+  A conventional HTTP status code:
+    * 401 - Authorization error with your API key or account
+    * 403 - Forbidden error with your API key or account
+    * 404 - The requested item does not exist
+    * 422 - The query or body parameters did not pass validation
+    * 429 - Too many requests have been sent with an API key in a given amount of time
```

Comment:
In the future, might be useful to link out to any formal guidance we provide publicly about rate limits.

Reply:
Good thought! I am a huge fan of contextual documentation. @nihwang will be looking at the spec from the docs point of view starting in Q1. Letting her know about your suggestion.